playbooks/roles/edxapp/templates/cms_gunicorn.py.j2

```diff
@@ -5,7 +5,7 @@ gunicorn configuration file: http://docs.gunicorn.org/en/develop/configure.html
 """
 import multiprocessing
 
-preload_app = True
+preload_app = False
```

Comment:
From the release notes: "To address a connection pooling issue in Ubuntu 16.04, we’ve changed how we launch new gunicorn web server processes."

Also gunicorn settings: http://docs.gunicorn.org/en/stable/settings.html#server-mechanics